source/fundamentals/data-formats/documents.txt

```diff
@@ -14,34 +14,379 @@ Documents
 Overview
 --------
 
-- Definition - what is a document, what format is it, where to find more info 
-- Table comparison of classes (columns for: implements Map, provides type safety, is an interface or class, compatibility with server versions
-- Recommend Document for general use and why.
-- Duplicate field name admonition (I forgot what this is)
+A MongoDB **document** is a data structure that contains key/value pairs in
+binary JSON (BSON) format. The document is the base unit of the MongoDB
+data hierarchy: each document belongs to a single collection
```

Comment:
> The document is the base unit of the MongoDB	data hierarchy

I feel like you're inventing new terminology here: I've never heard a document described this way in our documentation.  And it's not accurate.  The base unit is a BSON _value_ (e.g. int64, int32, boolean), not a BSON document.

And also, documents are used for other purposed besides storage in collections: e.g. commands, queries, sort criteria, etc

Reply:
Good point, I'll remove the phrase that mentions it's the base unit.